llm-lora-finetuning/utils/loaders.py

```diff
@@ -0,0 +1,120 @@
+# Apache Software License 2.0
+#
+# Copyright (c) ZenML GmbH 2024. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from functools import partial
+from pathlib import Path
+from typing import Any
+
+import torch
+from datasets import load_dataset
+from peft import LoraConfig, get_peft_model, prepare_model_for_kbit_training
+from transformers import AutoModelForCausalLM, BitsAndBytesConfig
+from utils.logging import print_trainable_parameters
+from utils.tokenizer import generate_and_tokenize_prompt, load_tokenizer
+
+
+def prepare_data(
+    base_model_id: str,
+    system_prompt: str,
+    is_tain: bool = True,
```

Comment:
is_train (and fixes elsewhere as a result?)

Reply:
This method was not used - I used step to do it, so just removed the whole thing